ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-release-2.5.yaml

```diff
@@ -101,6 +101,12 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ocm-ci-image-mirror
+- as: fast-forward
```

Comment:
only main branch needs ff. release branch doesnot need ff. 
remove this.

Reply:
Done.